README.md

```diff
@@ -21,6 +21,7 @@ Please find a list of the branches below, with more information on which kind of
 | [`run/from-0.67-to-0.69`](https://github.com/react-native-community/RNNewArchitectureApp/tree/run/from-0.67-to-0.69) | A full migration from RN 0.67 to 0.69 | ✅ | ✅ |
 | [`run/from-0.67-to-0.70`](https://github.com/react-native-community/RNNewArchitectureApp/tree/run/from-0.67-to-0.70) | A full migration from RN 0.67 to 0.70 | ✅ | ✅ |
 | [`run/pure-cxx-module`](https://github.com/react-native-community/RNNewArchitectureApp/tree/run/pure-cxx-module) | A step-by-step guide on how to integrate a Pure C++ module in a React Native 0.71.0-RC.2 app. Thanks to [@christophpurrer](https://github.com/christophpurrer) for the guide | ✅ | ✅ |
+| [`view-flattening-ios`](https://github.com/sammy-SC/RNNewArchitectureApp/tree/view-flattening-ios) | Examples of View Flattening on iOS  | | ✅ |
```

Comment:
do we have any docs explaining what view flattening is anywhere? What about Android?

Reply:
View Flattening doc: https://reactnative.dev/architecture/view-flattening
This has been developed specifically for Android and it was never documented for iOS.

Reply:
@kelset I also added https://github.com/reactwg/react-native-new-architecture/discussions/110
which has specific examples and links to this sample app.

Reply:
Android has had view flattening since the beginning. New Architecture brings it to iOS. 
I'll mention this in the working group post and in the sample app.

Reply:
```suggestion
| [`view-flattening-ios`](https://github.com/react-native-community/RNNewArchitectureApp/tree/example/view-flattening) | Examples of View Flattening on iOS  | | ✅ |
```

See my other comment for the reason.